Code/GraphMol/FileParsers/FileParsers.h

```diff
@@ -160,6 +160,19 @@ inline void MolToV3KMolFile(const ROMol &mol, const std::string &fName,
   MolToMolFile(mol, fName, includeStereo, confId, kekulize, true);
 }
 
+RDKIT_FILEPARSERS_EXPORT void MolToCMLBlock(std::ostream &os, const ROMol &mol,
+                                            int confId = -1,
+                                            bool kekulize = true);
```

Comment:
Please just remove this from the public API. It's fine to have the function in `CMLWriter.cpp` and use it there, but there's no need to expose it more broadly.

```suggestion
```